js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
```

Comment:
Is it really necessary to make separate views for all these sections? I'm all for modularity, but unless you think these sections will be re-used somewhere else in the app (which I really doubt), I think it's very much overkill here. I would make this all one Advanced view (look at Page.js as an example).

When you toJSON() the settings or localSettings model, it will include and flatten any nested attributes, so they could easily be handled in one view. Similary, getFormData() will properly handle nested attributes for you as long as you use the proper naming convention (namely giving them a flattened name attribute, e.g. `name="nestedAttribute.attributeName"`.